src/resources/CaseAssistConfigs/tests/CaseAssistConfig.spec.ts

```diff
@@ -60,13 +60,13 @@ describe('CaseAssistConfig', () => {
             },
             classificationConfigurations: [
                 {
-                    filter: 'filter_string',
+                    modelId: 'model-id',
                     fieldsToPredict: [
                         {
                             name: 'field1',
                         },
                     ],
-                    strategy: CaseClassificationStrategies.Some,
+                    strategy: CaseClassificationStrategies.Axon,
```

Comment:
Why don't you instead add one more config in your tests cases ?

Reply:
Because I don't think it's necessary.

Reply:
Oki 🤷🏻